astropy/wcs/tests/test_wcs.py

```diff
@@ -1214,7 +1223,14 @@ def test_footprint_contains():
     hasCoord = test_wcs.footprint_contains(SkyCoord(240, 2, unit='deg'))
     assert not hasCoord
 
-    hasCoord = test_wcs.footprint_contains(SkyCoord(24, 2, unit='deg'))
+    if sys.platform.startswith('win'):
```

Comment:
@nden , is this expected?

Reply:
Here ``np.errstate`` might make more sense (without the platform conditional)

Reply:
I don't have access to WIN but I wonder if [this](https://github.com/numpy/numpy/issues/8945#issuecomment-550288279) is the cause.

Reply:
Yes, surely that's it!

Given that it is therefore an upstream issue, maybe for now the best is to stick with a windows-specific `xfail`? At least then when it is fixed we'll notice and can remove it.

Reply:
Nice find, @nden. Thanks!